test/e2e/view-offender-manager-overview.js

```diff
@@ -9,7 +9,8 @@ describe('View overview', function () {
   before(function () {
     return dataHelper.selectIdsForWorkloadOwner()
       .then(function (results) {
-        workloadOwnerId = results.filter((item) => item.table === 'workload_owner')[0].id
+        var workloadOwnerIds = results.filter((item) => item.table === 'workload_owner')
+        workloadOwnerId = workloadOwnerIds[workloadOwnerIds.length - 1].id
```

Comment:
It actually doesn't matter, there should only be one. Nice catch, I'll revert because it's confusing.